.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Description
+
+Please provide a link to the related issue for the changes in this pull request.
+
+### Stability
+Keep in mind, there is a big userbase for lol-html so even small changes to API behavior can be
+breaking for some of them, and will be considered breaking for lol-html.
+
+**Lol-html has a commitment to stability; breaking changes are only to be considered as a last resort.**
+
+- [ ] My changes **do not** introduce any breaking changes or change existing API behaviors.
+- [ ] My changes **do** introduce breaking changes or changes existing API behaviors.
+  - [ ] I've discussed these changes with the repository maintainers and received a green light.
```

Comment:
I think we can just have one tick with:
> My changes **do** introduce breaking changes or changes existing API behaviors. I've discussed these changes with the repository maintainers and received a green light.

And make people tick it if that's the case.

Also, let's revive ticks for testing and documentation - those were useful. Another tick that would be useful is a reminder to update C API if changes to Rust API were made